addons/binding/org.openhab.binding.s7/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,23 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: S7 Binding
+Bundle-SymbolicName: org.openhab.binding.s7;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.2.0
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: ., lib/Moka7-project.jar
+Import-Package: 
+ com.google.common.collect;version="10.0.1",
```

Comment:
Do not use version constraints here, can you also check whether you could rewrite the code to not depend on this dependency?